ocs_ci/ocs/resources/pod.py

```diff
@@ -1216,24 +1216,39 @@ def get_plugin_pods(interface, namespace=None):
     return plugin_pods
 
 
-def plugin_provisioner_leader(interface, namespace=None):
+def plugin_provisioner_leader(
```

Comment:
I think the function name should reflect that the function purpose is to find the plugin_provisioner_leader.
Maybe `def find_plugin_provisioner_leader`?

Reply:
@ebenahar Shall we take it in a different PR? Existing usages should be modified.

Reply:
@ebenahar Updated the function name to get_plugin_provisioner_leader